app-center-webapps/src/main/webapp/vueLocales/locale_en.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Can you add a reference to this  file in  translations.properties file a root folder of the addon ?
So that, this file can be included in crowdin, and then translated in all languages ?

Thanks

Reply:
Okay